sprangu-front/src/app/components/filtering-component/filtering-component.component.html

```diff
@@ -0,0 +1,48 @@
+<div style="margin-top: 25px;">
```

Comment:
neišeitų be inline stilių perdaryt? Jeigu ką gali pakurti naują taską, galėsiu sutvarkyti, jeigu frontas ne prie širdies

Reply:
fixed, padariau su bootstrap + css 1 vietoj